tracing/tracer_test.go

```diff
@@ -67,34 +62,33 @@ func TestInitGlobalTracer_Disabled(t *testing.T) {
 }
 
 func TestInitGlobalTracer_NoServiceName(t *testing.T) {
-	tracer, closer, err := InitGlobalTracer(_emptyJaegerConfig, "", getLogger(), _statsReporter)
+	tracer, closer, err := InitGlobalTracer(_emptyJaegerConfig, "", zap.L(), _statsReporter)
 	assert.Error(t, err)
 	assert.Nil(t, tracer)
 	assert.Nil(t, closer)
 }
 
 func TestLoadAppConfig(t *testing.T) {
-	jConfig := loadAppConfig(_emptyJaegerConfig, getLogger())
+	jConfig := loadAppConfig(_emptyJaegerConfig, zap.L())
 	assert.NotNil(t, jConfig)
 	assert.NotNil(t, jConfig.Logger)
 }
 
 func TestLoadAppConfig_JaegerConfigWithLogger(t *testing.T) {
-	jConfig := loadAppConfig(_jaegerConfigWithLogger, getLogger())
+	jConfig := loadAppConfig(_jaegerConfigWithLogger, zap.L())
 	assert.NotNil(t, jConfig)
 	assert.Equal(t, jaeger.NullLogger, jConfig.Logger)
 }
 
 func TestLoadAppConfig_NilJaegerConfig(t *testing.T) {
-	jConfig := loadAppConfig(nil, getLogger())
+	jConfig := loadAppConfig(nil, zap.L())
```

Comment:
zap.L and zap.S are a bit confusing. Not blocking this PR, but we may need some verbose method here.